mapviz_plugins/src/textured_marker_plugin.cpp

```diff
@@ -444,13 +472,16 @@ namespace mapviz_plugins
   {
     ros::Time now = ros::Time::now();
 
+    float _alphaVal = alphaVal; // Set all markers to same alpha value
```

Comment:
Why make a copy of this member variable here?

Reply:
I wanted to copy the current value (at draw time) of the Alpha slider before iterating through the markers.  This should ensure that each marker in the list is set to the same alpha value for every draw step.

Reply:
If the concern is that this value could be modified while `Draw` is running, as it is right now, I think that shouldn't be a concern; Qt slots are always called from Qt's main thread, and drawing is also always done on the main thread.  It looks like `alphaVal` is only being modified when the `SetAlphaLevel` slot is called, so it should never be modified at the same time that this drawing is happening.

Although there's also no harm in leaving this as-is in case it could be modified by another thread in the future.